corehq/apps/dashboard/models.py

```diff
@@ -329,17 +329,14 @@ def _get_app_url(app):
             )
 
         def _get_view_app_url(app):
-            return reverse('view_app', args=[self.request.domain, app['id']])
+            return reverse('view_app', args=[self.request.domain, app.get_id])
```

Comment:
That's fair, haven't thought about it for years and I usually just go with "avoid pycharm's squigglies" by default. I'm (easily) convinced by your comment and would be happy to adopt that